### ISSUE TITLE
Multiple results for defining ontology search on UBERON term

### ISSUE THREAD

**Issue by mcourtot** [EBISPOT/OLS#326]
Hi,

We get multiple results for a query on is defining ontology for https://www.ebi.ac.uk/ols/api/terms/findByIdAndIsDefiningOntology?id=UBERON_0001037

We get UBERON as expected, but also the HCA ontology, which seems wrong. This creates issues with our validation script and is blocking one of our users from submitting.

Can you help?

cc @ke4

**Comment by simonjupp** [comment]
We need to remove the following values from the`baseUris` field in the HCAO config  https://www.ebi.ac.uk/ols/api/ontologies/hcao

* http://purl.obolibrary.org/obo/HsapDv_
* http://purl.obolibrary.org/obo/CL_ 
* http://purl.obolibrary.org/obo/UBERON_

**Comment by ke4** [comment]
Could you tell me what is the ETA of this fix, please?
HCA and IMAGE projects are blocked related to this issue.
Many thanks,
Karoly

**Comment by henrietteharmse** [comment]
@ke4 I did do the configuration change on prod already last week after which a re-index of HCAO must forced. Once the current data update is complete, I will do force for HCAO. It should be done by Wed, latest Friday.

**Comment by mcourtot** [comment]
Thanks @henrietteharmse! Can you please let us know when done? HCA is on hold for submission and complaining. 
Let us know if there is anyhting we can do to help on our side too.

**Comment by henrietteharmse** [comment]
A brief recap of the discussion with Melanie and Tony.

The behaviour as described by Melanie is indeed the intended behaviour of OLS. However, this behaviour is dependent on the configuration of the various ontologies in OLS, many of which OntoTools have no control over. This ideal behaviour could be broken, for example, in the following 2 ways:
(1 ) if 2 or more ontologies define the same base uri, or
(2) different base uri's are defined for the same ontology.
Neither of these cases can be detected by the OLS indexing process efficiently, because both could also be valid configurations. 

Tony suggested that services using OLS should be resilient even when multiple defining ontologies are returned for a term. 

This particular instance of this problem has been corrected, but there is no guarantee this will not happen again.
